NuRadioMC/utilities/Veff.py

```diff
@@ -204,6 +208,20 @@ def get_Veff_Aeff_single(filename, trigger_names, trigger_names_dict, trigger_co
         thetamin = fin.attrs['thetamin']
     if('thetamax' in fin.attrs):
         thetamax = fin.attrs['thetamax']
+
+    # restrict the theta range, if requested
+    theta_width_file = abs(np.cos(thetamin) - np.cos(thetamax))
+    if min(bounds_theta) > thetamin:
+        logger.info("restricting thetamin from sim file from {} to {}".format(thetamin, min(bounds_theta)))
+        thetamin = min(bounds_theta)
```

Comment:
So here you change the thetamin variable from the value stored in the file, to the value specified in the argument right? Is it okay to use the same variable name here, or should it be something like thetamin_attr before? But maybe its fine and Im just overcritical ;)

Reply:
ok, I don't think thetamin_attr makes it more readable, since this is the default value. Added a bit more verbose comments though